Integrations/Cybereason/Cybereason.py

```diff
@@ -0,0 +1,1477 @@
+    {'field': 'ownerMachine', 'header': 'Owner Machine', 'type': 'element'},
+    {'field': 'calculatedUser', 'header': 'User', 'type': 'element'},
+    {'field': 'imageFile', 'header': 'Image File', 'type': 'element'},
+    {'field': 'imageFile.sha1String', 'header': 'SHA1', 'type': 'simple'},
+    {'field': 'imageFile.md5String', 'header': 'MD5', 'type': 'simple'},
+    {'field': 'imageFile.companyName', 'header': 'Company Name', 'type': 'simple'},
+    {'field': 'imageFile.productName', 'header': 'Product Name', 'type': 'simple'}
+]
+
+PROCESS_FIELDS = [element['field'] for element in PROCESS_INFO]
+
+PROCESS_HEADERS = [element['header'] for element in PROCESS_INFO]
+
+CONNECTION_INFO = [
+    {'field': 'elementDisplayName', 'header': 'Name', 'type': 'simple'},
+    {'field': 'direction', 'header': 'Direction', 'type': 'simple'},
+    {'field': 'serverAddress', 'header': 'Server Address', 'type': 'simple'},
+    {'field': 'serverPort', 'header': 'Server Port', 'type': 'simple'},
+    {'field': 'portType', 'header': 'Port Type', 'type': 'simple'},
+    {'field': 'aggregatedReceivedBytesCount', 'header': 'Received Bytes', 'type': 'simple'},
+    {'field': 'aggregatedTransmittedBytesCount', 'header': 'Transmitted Bytes', 'type': 'simple'},
+    {'field': 'remoteAddressCountryName', 'header': 'Remote Country', 'type': 'simple'},
+    {'field': 'ownerMachine', 'header': 'Owner Machine', 'type': 'element'},
+    {'field': 'ownerProcess', 'header': 'Owner Process', 'type': 'element'},
+    {'field': 'calculatedCreationTime', 'header': 'Creation Time', 'type': 'time'},
+    {'field': 'endTime', 'header': 'End Time', 'type': 'time'}
+]
+
+CONNECTION_FIELDS = [element['field'] for element in CONNECTION_INFO]
+
+CONNECTION_HEADERS = [element['header'] for element in CONNECTION_INFO]
+
+HEADERS = {
+    'Content-Type': 'application/json',
+    'Connection': 'close'
+}
+
+''' HELPER FUNCTIONS '''
+
+
+def build_query(query_fields, path, template_context='SPECIFIC'):
+
+    limit = demisto.args().get('limit')
+    results_limit = int(limit) if limit else 10000
+    group_limit = int(limit) if limit else 100
+
+    query = {
+        'customFields': query_fields,
+        'perFeatureLimit': 100,
+        'perGroupLimit': group_limit,
+        'queryPath': path,
+        'queryTimeout': 120000,
+        'templateContext': template_context,
+        'totalResultLimit': results_limit
+    }
+
+    return query
+
+
+def http_request(method, url_suffix, data=None, json=None, headers=HEADERS):
+    LOG('running request with url=%s' % (SERVER + url_suffix))
+    try:
+        res = session.request(
+            method,
+            SERVER + url_suffix,
+            headers=headers,
+            data=data,
+            json=json,
+            verify=USE_SSL
+        )
+        if res.status_code not in {200, 204}:
+            return_error('Your request failed with the following error: ' + res.content + str(res.status_code))
+    except Exception, e:
```

Comment:
no need to catch exceptions here since they will be handled in the main (code doesn't do anything special with the error).